src/seedu/addressbook/data/person/Address.java

```diff
@@ -57,3 +75,51 @@ public boolean isPrivate() {
         return isPrivate;
     }
 }
+
+class Block {
+	private String _value;
+	
+	public Block(String value) {
+		_value = value;
+	}
+
+	public String getValue() {
+		return _value;
+	}
+}
+
+class Street {
+private String _value;
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.